libs/prebuilt/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -254,6 +260,12 @@ def _validate_chat_history(
     raise ValueError(error_message)
 
 
+PreModelHook = Union[
+    Callable[[StateSchema], StateSchema],
+    Runnable[StateSchema, StateSchema],
```

Comment:
async func support

Reply:
yep, will add (should be already supported out of box, but need a type annotation)